pandas/io/pytables.py

```diff
@@ -4333,9 +4338,15 @@ def read(
             if values.ndim == 1 and isinstance(values, np.ndarray):
                 values = values.reshape((1, values.shape[0]))
 
-            block = make_block(values, placement=np.arange(len(cols_)), ndim=2)
-            mgr = BlockManager([block], [cols_, index_])
-            frames.append(DataFrame(mgr))
+            if isinstance(values, np.ndarray):
```

Comment:
I would merge this with the line above

Reply:
i dont understand the suggestion

Reply:
put this

```
if values.ndim == 1 and isinstance(values, np.ndarray):
                values = values.reshape((1, values.shape[0]))	                
```
inside this if

Reply:
the purpose of that check is orthogonal to the purpose of this check.  4341-4348 are logically grouped together

Reply:
ok, I don't particularly like the fact that we have to construct things like this, but i guess ok